tests/sympc/protocol/aby3/aby3_test.py

```diff
@@ -83,3 +85,105 @@ def test_invalid_mpc_pointer(get_clients) -> None:
     # passing sharetensor pointer
     with pytest.raises(ValueError):
         ABY3.truncate(x, session, 2 ** 32, None)
+
+
+@pytest.mark.parametrize("security_type", ["semi-honest", "malicious"])
+def test_bit_injection_prime(get_clients, security_type) -> None:
+    parties = get_clients(3)
+    falcon = Falcon(security_type=security_type)
+    session = Session(parties=parties, protocol=falcon)
+    SessionManager.setup_mpc(session)
+    ring_size = PRIME_NUMBER
+
+    bin_sh = torch.tensor([[1, 1], [0, 0]], dtype=torch.bool)
+
+    shares = [bin_sh, bin_sh, bin_sh]  # All possible combinations
```

Comment:
Hmmm, how are these all possible combinations?

Reply:
My mistake , I forgot to remove the comment,In previous commits, I did an exhaustive tests,It really slowed down the CI, so modified the tests, to include all the edge cases in one.

Reply:
https://github.com/OpenMined/SyMPC/pull/266/commits/903e0bdbecd79930f28f0c07d2ed45a48768304e#diff-0de61ae0880dba790984703ffe467b17bd550abec01dc758b826253d5202cfc2L118

Reply:
The exhaustive changes before.